SpatialGDK/Source/SpatialGDKServices/Private/EditorWorkerController.cpp

```diff
@@ -1,16 +1,10 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
-#pragma once
+#include "EditorWorkerController.h"
 
-#if WITH_EDITOR
+#include "Private/SpatialGDKServicesPrivate.h"
 
-#include "Editor.h"
-#include "Modules/ModuleManager.h"
-#include "Settings/LevelEditorPlaySettings.h"
-#include "SpatialGDKEditorToolbar.h"
-
-namespace SpatialGDK
+namespace
```

Comment:
This will need the #if WITH_EDITOR wrapper

Reply:
Wrapped in #if WITH_EDITOR.